kubernetes-pipeline/values.yaml

```diff
@@ -35,26 +37,40 @@ jenkins:
 #        hosts:
 #          - example.com
 
+# This section is used to configure access to the docker repository where Jenkins will push images to.
 registry:
   server: 'https://index.docker.io/v1/'
   username: <REGISTRY_USERNAME>
   password: <REGISTRY_PASSWORD>
   email: <REGISTRY_EMAIL>
   address: index.docker.io
 
+# Github credentials
+# An access token could be used instead of the password for controlling access.
 github:
   username: <GITHUB_USERNAME>
   password: <GITHUB_PASSWORD>
 
 # Values for Spinnaker chart
 spinnaker:
+# Configure your Docker registries here to be used by Spinnaker
```

Comment:
@Aaquiff shall we at least provide a link to the Spinnaker `values.yaml` description README documentation, which may contain the explanations for each of these parameters?

Reply:
Added the link to https://github.com/helm/charts/blob/master/stable/spinnaker/values.yaml